pyccel/codegen/printing/fcode.py

```diff
@@ -465,11 +465,7 @@ def _print_Print(self, expr):
 
     def _print_SymbolicPrint(self, expr):
         # for every expression we will generate a print
-        def _iprint(e):
-            return "print *, 'sympy> {}'".format(e)
-        code = ''
-        for a in expr.expr:
-            code = '{code}{p}'.format(code=code, p=_iprint(a))
```

Comment:
So is this loop not needed at all? Why is that?

Reply:
Because I mis-typed my correction :/